src/blas/impl/KokkosBlas1_iamax_impl.hpp

```diff
@@ -63,7 +63,7 @@ struct V_Iamax_Functor
   typedef MagType                                                  mag_type;
   typedef typename XV::non_const_value_type                        xvalue_type;
   typedef Kokkos::Details::InnerProductSpaceTraits<xvalue_type>    IPT;
-  typedef typename Kokkos::MaxLoc<mag_type,size_type>::value_type  maxloc_type;
+  typedef typename RV::value_type                                  value_type;
```

Comment:
Prefer `using` alias syntax for new code.

Reply:
@mhoemmen ```using``` is used instead of ```typedef``` now.